server/odc-service/src/main/java/com/oceanbase/odc/service/iam/model/User.java

```diff
@@ -147,6 +154,24 @@ public User(UserEntity userEntity) {
         this.extraProperties = userEntity.getExtraPropertiesJson();
     }
 
+    public static UserEntity autoCreatedUserEntity(@NonNull String account, @NonNull String name,
```

Comment:
why do we put this method into `User`? It seems like it is not related to `User` at all.

Reply:
Avoid the same codes appear several times. Build user entity in User class is readable.

Reply:
Moving this method into the `UserEntity` makes more sense intuitively.

Reply:
Okay~